src/main/scala/nl.knaw.dans.easy.deposit/StateManager.scala

```diff
@@ -115,31 +119,57 @@ case class StateManager(depositDir: File, submitBase: File, easyHome: URL) exten
     UUID.fromString(draftProps.getString(bagIdKey))
   }
 
-  private def landingPage = {
-    Try { getProp("identifier.fedora", submittedProps) }
+  private def mailToDansMessage: String = {
+
+    val title: String = (draftDeposit.getDatasetMetadata.map(_.titles) match {
+      case Success(Some(titles: Seq[String])) => titles.headOption
+      case _ => None
+    }).getOrElse("").trim()
+    val shortTitle = title.substring(0, Math.min(42, title.length)).replaceAll("[\r\n]+", " ")
+    val mediumTitle = title.substring(0, Math.min(1000, title.length))
+    val ref = getSubmittedBagId.map(_.toString).getOrElse(s"DRAFT/$relativeDraftDir")
+    val subject = queryPartEncode(s"${ shortTitle } (reference nr: $ref)")
+    val body = queryPartEncode(
+      s"""Hello
+         |
+         |Could you please figure out what went wrong with my deposit?
+         |
+         |It has reference:
+         |   $ref
+         |and title:
+         |   $mediumTitle
+         |""".stripMargin
+    )
+    s"""Something went wrong while processing this deposit. Please <a href="mailto:info@dans.knaw.nl?subject=$subject&body=$body">contact DANS</a>"""
```

Comment:
Not sure if my concern makes any sense, but should we keep this address hardcoded? (1) scrapers could find this address and start sending spam to it, (2) if we want to change the address later, it is easier to do so if it is configurable.

Reply:
Looked into that. Found only existing hard-coded examples in our code, but not sure my search-result was complete.